opendebates/models.py

```diff
@@ -32,6 +33,11 @@ class Meta:
 
 
 class SiteMode(CachingMixin, models.Model):
+    THEME_CHOICES = [(theme, theme) for theme in settings.SITE_THEMES]
+
+    domain = models.CharField(_('domain name'), unique=True, max_length=100)
+    theme = models.CharField(max_length=255, choices=THEME_CHOICES)
```

Comment:
Is this+SITE_THEMES really doing anything for us anymore?  I *think* the only thing that `theme` is providing now is an automatic css inclusion from the codebase.  Now that we have `mode. inline_css` maybe we should just get rid of that altogether instead of having this awkward dependency on code state?

(But maybe that ought to be a separate PR/thread.)